src/mobx.ts

```diff
@@ -47,6 +47,7 @@ export { expr                                                 } from "./api/expr
 export { toJSON, toJS                                         } from "./api/tojs";
 export { ITransformer, createTransformer                      } from "./api/createtransformer";
 export { whyRun                                               } from "./api/whyrun";
+export { dispose                                              } from "./api/dispose";
```

Comment:
The name is a bit generic, maybe `disposeCurrentReaction`? That also makes it more clear that it should be called from inside a reaction